### DISCUSSION TITLE
Using :e %:h\filename then :w opens save as dialog

### DISCUSSION THREAD

**Discussion by ChimpSpin** [vscode-neovim/vscode-neovim#2336] [Q&A] [answered]
Is it possible to create the file without having to deal with the dialog?

**Comment by justinmk** [comment, marked as answer]
Open to ideas. You can find the reason for the current behavior https://github.com/vscode-neovim/vscode-neovim/issues/1260#issuecomment-2148445015 , it's the least-bad solution we could find.
